services/fakes/fake_softlayer_product_package.go

```diff
@@ -24,7 +26,24 @@ func (fps *FakeProductPackageService) GetItemsByType(packageType string) ([]data
 }
 
 func (fps *FakeProductPackageService) GetItemPrices(packageId int, filters string) ([]datatypes.SoftLayer_Product_Item_Price, error) {
-	return []datatypes.SoftLayer_Product_Item_Price{}, errors.New("Not supported")
+	var textFixtrue = ""
+
+	switch {
+	case strings.Contains(filters, "keyName"):
+		textFixtrue = "SoftLayer_Product_Package_getItemPrices.json"
+	case strings.Contains(filters, "100") || strings.Contains(filters, "1000") || strings.Contains(filters, "2000"):
+		textFixtrue = "SoftLayer_Product_Package_getItemPricesBySizeAndIops.json"
+	}
+
+	if textFixtrue == "" {
+		return []datatypes.SoftLayer_Product_Item_Price{}, errors.New(fmt.Sprintf("No matched IOPS found for filters %s", filters))
+	}
+
+	response, _ := testhelpers.ReadJsonTestFixtures("services", textFixtrue)
```

Comment:
I referenced [this function](https://github.com/maximilien/softlayer-go/blob/master/services/fakes/fake_softlayer_product_package.go#L17-L24) to write my code. I think I could use another way to fake functions in [softlayer_product_package.go](https://github.com/maximilien/softlayer-go/blob/master/services/softlayer_product_package.go)